packages/server/routes/api/events.js

```diff
@@ -117,26 +117,23 @@ router.get(
 router.get(
   "/:eventId/attendees",
   AuthController.authenticate,
+  [
+    param("eventId")
+      .trim()
+      .blacklist("<>")
+      .isUUID()
+      .withMessage((eventId) => `Given eventId ${eventId} is invalid.`)
+      .bail()
+      .custom(async (eventId) => {
+        if (!(await EventController.existsInPrisma(eventId))) {
+          throw new Error(`No event exists with an ID of ${eventId}`);
+        }
+      }),
+  ],
   async (request, response) => {
-    const eventId = request.params.eventId;
-
-    if (!validate(eventId)) {
-      response.status(400).json({
-        status: "fail",
-        data: {
-          eventId: `Given eventId ${eventId} is invalid.`,
-        },
-      });
-      return;
-    }
-    if (!(await EventController.existsInPrisma(eventId))) {
-      response.status(404).json({
-        status: "fail",
-        data: {
-          eventId: `No event exists with an ID of ${eventId}`,
-        },
-      });
-      return;
+    const errors = validationResult(request);
+    if (!errors.isEmpty()) {
+      return response.status(400).json({ errors: errors.array() });
     }
     try {
       const { eventId } = request.params;
```

Comment:
Since we're using express-validator now, we need to import `matchedData()` to get the validated data from express-validator if it passes. Don't use `request.params` and `request.body` anymore.

```js
const { eventId } = matchedData(request);
```